android/app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,8 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.READ_PROFILE" />
+    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
```

Comment:
This require comment too.

Reply:
This is required for the notifications 
![image](https://user-images.githubusercontent.com/13368542/40554419-7a389bc2-5ffa-11e8-9b0d-6b054e1daf0e.png)